README.md

```diff
@@ -1017,6 +1018,7 @@ For example, to connect with `kafkacat` you could run:
     kafkacat -b my-cluster-kafka-bootstrap:9092 -X security.protocol=SASL_PLAINTEXT -X sasl.mechanism=PLAIN -X sasl.username=team-a-client -X sasl.password="team-a-client-secret" -t a_topic -P
 
 
+
```

Comment:
Do we need the triple empty line? Wasn't double empty line enough? (Not a big deal, feel free to keep it if you think it is more readable or something).